src/bitmessageqt/statusbar.py

```diff
@@ -13,6 +17,7 @@ def __init__(self, parent=None):
         self.iterator = 0
 
     def timerEvent(self, event):
+        """It indicates timer event."""
```

Comment:
this is an event handler, not an indicator. It allows to queue and prioritise messages to show in the status bar, for example if many messages come very quickly after one another, it adds delays and so on